unlock-app/src/server.js

```diff
@@ -0,0 +1,36 @@
+const { createServer } = require('http')
+const { parse } = require('url')
+const next = require('next')
+const pathMatch = require('path-match')
+
+const port = parseInt(process.env.PORT, 10) || 3000
+const dev = process.env.NODE_ENV !== 'production'
+const app = next({ dir: './src', dev })
+const handle = app.getRequestHandler()
+const route = pathMatch()
+
+app.prepare()
+  .then(() => {
+    createServer((req, res) => {
+      const parsedUrl = parse(req.url, true)
+      const { pathname, query } = parsedUrl
+
+      // assigning `query` into the params means that we still
+      // get the query string passed to our application
+      const path = pathname.split('/')[1]
```

Comment:
It would be cool if we found a way to avoid writing the routing logic twice.

Reply:
Any ideas how you would like to see this being abstracted?

Reply:
I think we can get started with some kind of manifest file with the list of routes and how they're being handled.